makeflow/src/makeflow_viz.c

```diff
@@ -207,6 +235,42 @@ int main(int argc, char *argv[])
 			case LONG_OPT_DOT_FILE:
 				file_attr = xxstrdup(optarg);
 				break;
+			case LONG_OPT_JSON:
+				dag_syntax = DAG_SYNTAX_JSON;
+				break;
+			case LONG_OPT_JX:
+				dag_syntax = DAG_SYNTAX_JX;
+				break;
+			case LONG_OPT_JX_ARGS:
```

Comment:
It might be better to refactor this stuff, since handling these options shows up a few times in this PR, and it's also in `jx2json`